### ISSUE TITLE
transfer learning of sq-ssd-lite

### ISSUE THREAD

**Issue by ynjiun** [qfgaohao/pytorch-ssd#146]
Hi, this is a great repository I am look for! Thanks a lot for your contribution.
I am trying to do a transfer learning from your sq-ssd-lite to learn only detection 3 different object types:
Question might need your help:
1. could you provide your sq-ssd-lite pretrained model and label URL to allow wget from?
2. could you give me a hint on how to set the num_classes = 3? that is, make a minor modification of pretrained network then retrain on 3 object types dataset?
Thank you so much for your help.
